### ISSUE TITLE
Markdown shown in the tN smurf bar

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord/translationCore#6600]
This was fine in 2.0.
Shows up markdown in 2.1: Hyperbole, Abstract Noun and few other examples.
![image.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/159b5d80-205b-45d4-b971-33f53a4d1b85)

**Comment by PhotoNomad0** [comment]
@elsylambert - Thanks, this looks like a bug introduced with the conversion to react hooks.

**Comment by PhotoNomad0** [comment]
@elsylambert - ~looks like it is a bug when migrating from 2.0.0.  If I import a usfm titus project into 2.1.0, the quotes look fine - no markdown.~

**Comment by PhotoNomad0** [comment]
@elsylambert - looks like a resource change, they started using markdown in occurrenceNote.  For example in the old resource file at `translationCore/resources/en/translationHelps/translationNotes/v17/figures/groups/tit/figs-hyperbole.json` we have:
```
[
  {
    "comments": false,
    "reminders": false,
    "selections": false,
    "verseEdits": false,
    "nothingToSelect": false,
    "contextId": {
      "occurrenceNote": "“Cretans lie all the time.” This is an exaggeration that means many Cretans lie a lot. (See: [Hyperbole and Generalization](rc://en/ta/man/translate/figs-hyperbole))",
      "reference": {
        "bookId": "tit",
        "chapter": 1,
        "verse": 12
      },
      "tool": "translationNotes",
      "groupId": "figs-hyperbole",
      "quote": [
        {
          "word": "Κρῆτες",
          "occurrence": 1
        },
        {
          "word": "ἀεὶ",
          "occurrence": 1
        },
        {
          "word": "ψεῦσται",
          "occurrence": 1
        }
      ],
      "quoteString": "Κρῆτες ἀεὶ ψεῦσται",
      "glQuote": "Cretans are always liars",
      "occurrence": 1
    }
  }
]
```

But in `/Users/blm/translationCore/resources/en/translationHelps/translationNotes/v21/figures/groups/tit/figs-hyperbole.json` we have:
```
[
  {
    "comments": false,
    "reminders": false,
    "selections": false,
    "verseEdits": false,
    "nothingToSelect": false,
    "contextId": {
      "occurrenceNote": "***Cretans lie all the time.*** This is an exaggeration that means that Cretans had a reputation to be liars. (See: [Hyperbole](rc://en/ta/man/translate/figs-hyperbole))",
      "reference": {
        "bookId": "tit",
        "chapter": 1,
        "verse": 12
      },
      "tool": "translationNotes",
      "groupId": "figs-hyperbole",
      "quote": [
        {
          "word": "Κρῆτες",
          "occurrence": 1
        },
        {
          "word": "ἀεὶ",
          "occurrence": 1
        },
        {
          "word": "ψεῦσται",
          "occurrence": 1
        }
      ],
      "quoteString": "Κρῆτες ἀεὶ ψεῦσται",
      "glQuote": "Cretans are always liars",
      "occurrence": 1
    }
  }
]
```

**Comment by PhotoNomad0** [comment]
@elsylambert @cckozie - have a test build here to see if this fixes the markdown issue:

[Linux build 2.1.0 (9582c1f)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-linux-x64-2.1.0-9582c1f.deb) is ready for download.
[macOS build 2.1.0 (9582c1f)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-macos-x64-2.1.0-9582c1f.dmg) is ready for download.
[Windows x64 build 2.1.0 (9582c1f)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-win-x64-2.1.0-9582c1f.setup.exe) is ready for download.
[Windows x32 build 2.1.0 (9582c1f)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-win-x32-2.1.0-9582c1f.setup.exe) is ready for download.

**Comment by elsylambert** [comment]
@PhotoNomad0 2.1.0 (9582c1f). It shows html tags:
![image](https://user-images.githubusercontent.com/51806180/70657936-dff80900-1c2a-11ea-9eed-f4075f16a45a.png)
![image](https://user-images.githubusercontent.com/51806180/70657969-f9995080-1c2a-11ea-8a25-b8b9ec4dbe12.png)

**Comment by PhotoNomad0** [comment]
@elsylambert - Sorry, my goof. Didn't have everything in the build.  I started checking and got distracted.  Starting a rebuild now.

**Comment by PhotoNomad0** [comment]
@elsylambert - Second attempt here:

[Linux build 2.1.0 (e7d4a0e)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-linux-x64-2.1.0-e7d4a0e.deb) is ready for download.
[macOS build 2.1.0 (e7d4a0e)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-macos-x64-2.1.0-e7d4a0e.dmg) is ready for download.
[Windows x64 build 2.1.0 (e7d4a0e)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-win-x64-2.1.0-e7d4a0e.setup.exe) is ready for download.
[Windows x32 build 2.1.0 (e7d4a0e)](https://tc-builds.door43.org/build/feature-mcleanb-6600/translationCore-win-x32-2.1.0-e7d4a0e.setup.exe) is ready for download.

**Comment by elsylambert** [comment]
@PhotoNomad0 It has been fixed in all the places where it showed Markdown except I see it in one place in a Titus project:
![image](https://user-images.githubusercontent.com/51806180/70662736-dbd0e900-1c34-11ea-8b37-6888652a2574.png)

**Comment by PhotoNomad0** [comment]
@elsylambert - thanks, that looks like a content mistake - has quote at beginning and ends with ***.

**Comment by PhotoNomad0** [comment]
created a PR for content errors: https://git.door43.org/unfoldingWord/en_tn/pulls/1324

**Comment by PhotoNomad0** [comment]
Verified in Develop build translationCore 2.1.0 (3d944e3) _**this is not in release v2.1.0 **_
Develop builds here: http://tc-builds.door43.org.s3-website-us-west-2.amazonaws.com/build/develop/

**Comment by cckozie** [comment]
2.1.0 (ed1c32c) 
Most problems displaying markdown have been fixed, but this one remains. (Maybe cuz the closing tag is at the end?)

![image](https://user-images.githubusercontent.com/8171759/71482514-8b2ac580-27c8-11ea-9209-7d31c5f05e51.png)

From the catalog
![image](https://user-images.githubusercontent.com/8171759/71482535-aeee0b80-27c8-11ea-877c-1ae52524a269.png)

**Comment by PhotoNomad0** [comment]
@cckozie - this is strange, we had fixed the content for that check and Elsy verified.  But when I check on my projects, I have one that is right:

![Screen Shot 2019-12-28 at 5 54 19 AM](https://user-images.githubusercontent.com/14238574/71542664-1aee8200-2937-11ea-87f1-6ab27de966c7.png)

and two that don't even have the check:

![Screen Shot 2019-12-28 at 5 55 32 AM](https://user-images.githubusercontent.com/14238574/71542674-39547d80-2937-11ea-955b-f31423bccf2c.png)

On my system, for en translationNotes resources I have both version v21 and v22.  v21 has the invalid check you saw, and v22 has the fixed check.  

Looks like I have a mystery to figure out why the checks didn't update for a couple of my projects.

**Comment by PhotoNomad0** [comment]
@cckozie - OK found a logic error that projects don't always copy the latest tn checks into project.  Not related to what you saw, but a serious bug.

**Comment by PhotoNomad0** [comment]
@cckozie @elsylambert - to recreate the problem I see, update to the latest en and hi resources. then open this project: [en_alcz_tit_book.zip](https://github.com/unfoldingWord/translationCore/files/4009665/en_alcz_tit_book.zip).
This project only has titus 3:14 under `Double Negatives`, but latest resources have 4 verse.  On the tN card, deselect everything but `Figures of Speech/Double Negatives`.  When you launch with en GL it will only show one verse under `Double Negatives`.  I think the problem started because this imported project started with hi as GL initially.  But when GL is changed to en, the additional checks are not being copied into the menu.

**Comment by elsylambert** [comment]
@PhotoNomad0 with the given project and the latest resources, I saw only one verse for Double negatives tit3:14, where as there are three other verses in tN. 
This is written up  as a separate issue #6626

**Comment by elsylambert** [comment]
@PhotoNomad0  In my comment I had mentioned the same Category "Double negatives" showing markdown in the end as pointed out by @cckozie . It was noted as Content mistake.

**Comment by elsylambert** [comment]
> @PhotoNomad0 It has been fixed in all the places where it showed Markdown except I see it in one place in a Titus project:
> ![image](https://user-images.githubusercontent.com/51806180/70662736-dbd0e900-1c34-11ea-8b37-6888652a2574.png)

@PhotoNomad0 In my comment I had mentioned the same Category "Double negatives" showing markdown in the end as pointed out by @cckozie . 
2.1.0 (c6f260b)  I do not see Markdown anymore

**Comment by PhotoNomad0** [comment]
Should not see error with latest en resources.  Working on PR for issue @elsylambert created.

**Comment by cckozie** [comment]
2.1.0 (ed1c32c)
After downloading the latest en resources, I'm no longer seeing the problem in 2:15 under double negatives, but I am now seeing these two issues both of which appear to be content problems.
![image](https://user-images.githubusercontent.com/8171759/71604484-f0afe680-2b27-11ea-8c46-d5cfa09145e8.png)
![image](https://user-images.githubusercontent.com/8171759/71604924-df1c0e00-2b2a-11ea-8062-b795cc9adf75.png)


![image](https://user-images.githubusercontent.com/8171759/71604491-fb6a7b80-2b27-11ea-9f6e-72ab95cf0f89.png)
![image](https://user-images.githubusercontent.com/8171759/71604877-83518500-2b2a-11ea-979c-60e6e440aca6.png)

**Comment by PhotoNomad0** [comment]
@cckozie - did you pass these last to items on to the content team?

**Comment by elsylambert** [comment]
Seeing this issue in 2.1.0 Release in a Titus Project.

**Comment by birchamp** [comment]
Doublet -Titus 2:8 - TN shows This word ** has the same basic meaning as **uncorrupted** in 2:7. In 2:7, Paul states the meaning negatively: **uncorrupted**, meaning, **without error**, and in 2:8 he states the meaning positively: **sound, whole**, meaning **correct**. Both terms refer to Titus’s teaching. Use either positive or negative terms in the target language, or use one term with this meaning if it is difficult to use two terms

**Comment by cckozie** [comment]
From Larry:
@**Birch** I noticed a minor issue with tC while working on Titus today. The smurf bar in the tN tool does not handle the new bold formattting (and I assume, the bold-italics) in the notes.
![pasted_image](https://user-images.githubusercontent.com/8171759/75051360-5c149580-5493-11ea-9344-53e8e68344e7.png)

**Comment by cckozie** [comment]
This looks to be fixed in 2.2.0 (b6b7099)
![image](https://user-images.githubusercontent.com/8171759/75051484-954d0580-5493-11ea-8fa2-b489e6b2042e.png)

![image](https://user-images.githubusercontent.com/8171759/75051301-3dae9a00-5493-11ea-8f3c-e70e1a779ec0.png)

**Comment by elsylambert** [comment]
<img width="1472" alt="Screen Shot 2020-02-21 at 11 20 37 AM" src="https://user-images.githubusercontent.com/51806180/75051675-50799c80-549c-11ea-813e-764b06d0de80.png">
@cckozie Are you able to see this in titus project in 2.2?
Markdown seems to be working fine in 2.2.

**Comment by cckozie** [comment]
@elsylambert - That one is actually a content error. Here is that check in the catalog:
TIT	2	1	tpi2	figs-explicit	σὺ δὲ λάλει ἃ πρέπει	1	But you, speak what fits	q
It has been fixed in the latest catalog updates.

**Comment by cckozie** [comment]
Since these problems appear to be fixed in 2.2 I am closing this issue and will add regression scripts to test for them.
